ruby/lib/grpclb/client.rb

```diff
@@ -0,0 +1,56 @@
+require 'grpclb_balancer_v1/balancer_pb'
+require 'grpclb_balancer_v1/balancer_services_pb'
+
+class Grpclb::Client
```

Comment:
sooo, I prefer to implement this as a https://www.rubydoc.info/gems/grpc/GRPC/ClientInterceptor, then just use it to pass to our client stub, via `interceptors: [Grpclb::ClientInterceptor.new(lb_addr, target, max_reconnects: 3)]` option

Reply:
Still reading, but doesn't look like we can reconnect from it.

Reply:
Yes, it won't help for client reconnect.

Reply:
We can actually try to use `GRPC::ServerInterceptor` for load measuring, looks to be the proper way to do it + we can measure not only requests count, but also streamed items count, it seems...